docs/releases/1.9.txt

```diff
@@ -167,6 +167,12 @@ Backwards incompatible changes in 1.9
 
 ...
 
+Removed CSS related to IE6&7
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+All files related to IE6&7, specifcally CSS files and images, have been
+removed and their references with admin views have been deleted. Django
+1.9+ will longer support IE6&7.
```

Comment:
timgraham changed the wording (and the grammatical mistakes), you can see what was added in the commit here: https://github.com/django/django/commit/7a90b53d60f6a66340811417547e33e0bfadf09e